packages/node-cli/src/core/sdksetup/SDKDownloadService.js

```diff
@@ -34,19 +34,21 @@ class SDKDownloadService {
 		this._fileSystemService = new FileSystemService();
 	}
 
-	download() {
+	async download() {
```

Comment:
why does it need to be async?

Reply:
because we need to `await` for it in vscode-extension before executing any command